### DISCUSSION TITLE
Dark mode & CSS preprocessor

### DISCUSSION THREAD

**Discussion by Helveg** [executablebooks/meta#664] [Ideas]
I'm working on a dark mode for `sphinx-tabs` with the `data-theme` selector and `media(prefers-color-scheme: dark)` query;

If the CSS is going to increase in complexity, would anyone object to using a CSS preprocessor like [Sass](https://sass-lang.com/) to simplify things? We'd have to run the sass cli to compile the source file into the final static CSS file somewhere it makes sense.

**Comment by chrisjsewell** [comment]
Heya, I'd note https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html already supports dark mode

**Reply by chrisjsewell** [reply]
(it also has "first-class" support in furo: https://pradyunsg.me/furo/kitchen-sink/sphinx-design/)

**Comment by Helveg** [comment]
Oh, ok. Does that furo feature supercede this package then somehow? The PR wasn't much work 🤷 https://github.com/executablebooks/sphinx-tabs/pull/147
